types/src/key.rs

```diff
@@ -1358,6 +1394,7 @@ impl Distribution<Key> for Standard {
             17 => Key::AddressableEntity(rng.gen()),
             18 => Key::ByteCode(rng.gen()),
             19 => Key::Message(rng.gen()),
+            21 => Key::BlockMessageCount(BlockTime::new(rng.gen())),
```

Comment:
The `match` line for this needs updated (it currently has `match rng.gen_range(0..=18) {`).

Reply:
Added in https://github.com/casper-network/casper-node/pull/4550/commits/4550ba389d5e0871c090f901c4c5ac5bd29648fe